service/src/main/resources/db/changesets/20231208_clone-workflows-app-resources.yaml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+- changeSet:
+    id: Set cloneInstructions to COPY_RESOURCE for WORKFLOWS_APP resources.
+    author: mspector
+    changes:
+    - update:
+        tableName: resource
+        columns:
+            - column:
+                name: cloning_instructions
+                value: "COPY_RESOURCE"
+        where:
+          or:
+              and:
+                - column:
+                    name: resource_type
+                    value: "AZURE_DATABASE"
+                - column:
+                    name: name
+                    value: "cbas"
```

Comment:
It seems that this `and` condition would also update the cloning instructions of `cbas` database for `CROMWELL` app type. Should we add additional check to ensure it only does so for `WORKFLOWS_APP` app type?

Reply:
I'm not 100% sure if it matters, but I can add that check. better to be safe

Reply:
I think WSM might start cloning the `cbas` database in `CROMWELL` app if its set to `COPY_RESOURCE` since in WSM I don't think there is a check to see which app it belongs too. And if the user somehow in the workspace spun up a `CROMWELL` app then Leo will attach the cloned `cbas` database new app and I am not sure how things will work after this. So yeah I agree its better to be safe by adding that additional check.